packages/lib-utils/src/app/redux/actions/k8s.ts

```diff
@@ -304,9 +297,7 @@ export const watchK8sObject =
       return;
     }
 
-    const { subprotocols } = getImpersonate(getState()) || {};
-
-    WS[id] = k8sWatch(k8sType, queryWithCluster, { subProtocols: subprotocols }).onBulkMessage(
+    WS[id] = k8sWatch(k8sType, queryWithCluster).onBulkMessage(
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
@vidyanambiar since we're in this area, could you have a look and see if we could remove this Any? I think we added it temporarily and it would be nice to enforce the correct types.

Reply:
We have this same Any on a similar line above as well.

Reply:
@florkbr I've addressed this comment in the new commit. Please let me know if you have any suggestions for improving how I've implemented it. Thank you!